edit_topictagged_form.php

```diff
@@ -121,67 +114,34 @@ protected function definition_inner($mform) {
 				// count available questions
 				$value = 0;
 
+                global $sql_questionids_anytopic_anydifficulty;
+                global $sql_questionids_anydifficulty;
+                global $sql_questionids_anytopic;
+                global $sql_questionids;
+
+                require('consts.php');
```

Comment:
Same mention about making these variables harder to use as before. However, since the queries here appear to be reused, it is useful to move them. You could just have a function that handles the queries and even return a different type of result depending on what you need - a serialized representation of all records, or the number of records.